modules/ec2_instances/variables.tf

```diff
@@ -1,91 +1,120 @@
   description = "The AWS ami id to use"
 }
 
 variable "instance_type" {
-  default     = "t2.micro"
   description = "AWS instance type to use"
+  default     = "t2.micro"
 }
 
 variable "max_size" {
-  default     = 1
   description = "Maximum size of the nodes in the cluster"
+  default     = 1
 }
 
 variable "min_size" {
-  default     = 1
   description = "Minimum size of the nodes in the cluster"
+  default     = 1
 }
 
 #For more explenation see http://docs.aws.amazon.com/autoscaling/latest/userguide/WhatIsAutoScaling.html
 variable "desired_capacity" {
-  default     = 1
   description = "The desired capacity of the cluster"
+  default     = 1
 }
 
-variable "iam_instance_profile_id" {
-  description = "The id of the instance profile that should be used for the instances"
-}
-
-variable "private_subnet_ids" {
-  type        = "list"
-  description = "The list of private subnets to place the instances in"
-}
-
-variable "load_balancers" {
-  type        = "list"
-  default     = []
-  description = "The load balancers to couple to the instances. Only used when NOT using ALB"
+variable "instance_group" {
+  description = "The name of the instances that you consider as a group"
+  default     = "default"
 }
 
 variable "depends_id" {
   description = "Workaround to wait for the NAT gateway to finish before starting the instances"
 }
 
+#--------------------------------------------------------------
+# SSH settings
+#--------------------------------------------------------------
+
 variable "key_name" {
   description = "SSH key name to be used"
 }
 
-variable "custom_userdata" {
+variable "jump_ssh_sg_id" {
+  description = "jumpbox security group to allow access to the container runners"
   default     = ""
-  description = "Inject extra command in the instance template to be run on boot"
 }
 
-variable "ecs_config" {
-  default     = "echo '' > /etc/ecs/ecs.config"
-  description = "Specify ecs configuration or get it from S3. Example: aws s3 cp s3://some-bucket/ecs.config /etc/ecs/ecs.config"
-}
+#--------------------------------------------------------------
+# Logging
+#--------------------------------------------------------------
 
 variable "ecs_logging" {
-  default     = "[\"json-file\",\"awslogs\"]"
   description = "Adding logging option to ECS that the Docker containers can use. It is possible to add fluentd as well"
+  default     = "[\"json-file\",\"awslogs\"]"
 }
 
-variable "jump_ssh_sg_id" {
-  description = "jumpbox security group to allow access to the container runners"
+variable "cloudwatch_prefix" {
+  description = "If you want to avoid cloudwatch collision or you don't want to merge all logs to one log group specify a prefix"
   default     = ""
 }
 
+#--------------------------------------------------------------
+# Tags
+#--------------------------------------------------------------
+
 variable "owner" {}
+
+variable "cluster" {
+  description = "The name of the cluster"
+}
+
+variable "environment" {
```

Comment:
should this be workspace to meet tf standards?

Reply:
Done :)